db/migrate/20201021141827_clear_old_totats.rb

```diff
@@ -0,0 +1,18 @@
+class ClearOldTotats < Mongoid::Migration
+  # mirror of FeatureFlag.rb, so this migration won't error if that class is renamed/altered
+  class UserMigrator
+    include Mongoid::Document
+    store_in collection: 'users'
+    field :totat_t_ti, type: String
+    field :totat
+  end
+
+  def self.up
+    UserMigrator.where(:totat.exists => true).each do |user|
+      user.update(totat: nil, totat_t_ti: nil)
```

Comment:
Since `totat_t_ti` has been removed from the `User` model, we can't use it here in the migration.  It errors out and prevents downstream migrations from running.  If you want to remove the field from the document, you need to call `user.remove_attribute(:totat_t_ti)`.  That will fail gracefully if it doesn't exist.

Reply:
yeah, this has been updated to not use the User model anymore -- it's using the UserMigrator class, which is defined in this migration, and so we can guarantee it has the totat_t_ti field